site/components/pages/Redesign/index.tsx

```diff
@@ -60,6 +74,62 @@ export const Home: NextPage<Props> = (props) => {
           <NavItemMobile url={urls.stake} name="Wrap" />
           <NavItemMobile url={urls.bond} name="Bond" />
         </HeaderMobile>
+        <Section contentVariant="hero">
+          <Columns variant="hero">
+            <ContentBox variant="hero">
+              <Copy text="👋 WELCOME TO" />
+              <Heading text="KlimaDao" />
+              <Copy text="KlimaDAO harnesses the power of cryptocurrency, blockchain and smart contracts to create incentives for environmental protection." />
+              <ButtonPrimary
+                key="Enter App"
+                label="Enter App"
+                href={urls.app}
+              />
+            </ContentBox>
+            <ContentBoxImage variant="belowTextBox">
+              <Image alt="Intro" src={introPic} layout="fill" />
+            </ContentBoxImage>
+          </Columns>
+        </Section>
+        <Section variant="white" contentVariant="contained">
+          <Heading
+            align="center"
+            text="KlimaDAO’s vision is a future where the cost of carbon to the climate is embedded into our economic system."
+          />
+          <Columns variant="contained">
+            <Copy text="KlimaDAO is creating and governing a carbon-backed currency, which acts as the base unit of the on-chain carbon economy." />
+            <Copy text="Through the KLIMA token, we can help to align incentives between investors, civil society, and organizations, towards a more sustainable future." />
```

Comment:
wow no linebreaks anywhere... 💅 

these will all become Macros when we do i18n

```jsx
<Copy text={t`KlimaDAO is creating and governing a carbon-backed currency, which acts as the base unit of the on-chain carbon economy.`} />
```